docs/operate/config-reference.mdx

```diff
@@ -911,9 +951,9 @@ export HYP_DEFAULTSIGNER_*="...."
 </TabItem>
 
 <TabItem value="env" label="As Env">
-```bash
-export HYP_DEFAULTSIGNER_TYPE="hexKey"
-```
+  ```bash 
```

Comment:
also not a big deal and I don't think it needs to be changed in this PR, but wonder if we should be consistent with using bash vs sh, seems they're interspersed

Reply:
agree - changed to bash for consistency https://github.com/hyperlane-xyz/v3-docs/pull/305/commits/f9fec27cf3a56b11966a5eb44ca626e26ace5e20